src/Java/gtPlusPlus/core/item/ModItems.java

```diff
@@ -713,6 +713,7 @@ public static final void init(){
 			GT_OreDictUnificator.registerOre("platePhasedIron", ItemUtils.getSimpleStack(itemPlatePulsatingIron));
 		}
 		else {
+			new MaterialEIO();
```

Comment:
No. then this wont work as intended.

Reply:
Replaced with - https://github.com/draknyte1/GTplusplus/commit/e0738642c16a1f63a48bdd292e1275f6436126ce